packages/material-ui/src/styles/createSpacing.d.ts

```diff
@@ -1,4 +1,4 @@
-export type Spacing = (value: number | string) => number | string;
+export type Spacing = (...value: Array<number|string>) => number | string;
```

Comment:
This is to wide. Please see https://github.com/mui-org/material-ui/issues/14515#issuecomment-463115121 for more information.